install/linux/usr/share/odemis/hwtest/tmcm-can-ctrl.odm.yaml

```diff
@@ -0,0 +1,20 @@
+SimOptical: {
+    class: Microscope,
+    role: brightfield,
+    children: [Light, TMCM_CAN],
+}
+
+Light: {
+    class: simulated.Light,
+    role: brightlight,
+}
+
+TMCM_CAN: {
+    class: tmcm.CANController,
+    role: null,
+    init: {
+        channel: "can0", node_id: 1, datasheet: "TMCM-1240_CANopen_V322.dcf",
+        axes: {"x"}, refproc: "Standard", param_file: "test/tmcm-pd1240.tmcm.tsv",
+        ustepsize: [0.0001227184630308513],  # 2 * math.pi / (200 * 2 ** 8)
```

Comment:
Please, put each key on a separate line.
Also, add the unit as "rad". Otherwise a stepsize depending on PI sounds a little fancy ;-)